modules/nodes-containers-volumes-adding.adoc

```diff
@@ -37,11 +37,11 @@ character.
 |`'*'`
 
 |`-m, --mount-path`
-|Mount path inside the selected containers.
+|Mount path inside the selected containers. Do not mount to the container root, `/`, or any path that is the same in the host and the container, as this can corrupt your host system if the container is sufficiently privileged, such as the host `/dev/pts` files. It is safe to mount the host by using `/host`. 
```

Comment:
Maybe break this addition into three sentences?
```
Do not mount to the container root, `/`, or any path that is the same in the host
and the container. This can corrupt your host system if the container is sufficiently
privileged, such as the host `/dev/pts` files. It is safe to mount the host by using
`/host`.
```
Or maybe the use of a semi-colon to at least break up the long sentence :thinking: